index.d.ts

```diff
@@ -100,8 +95,15 @@ export namespace Cmd {
   export type GetState = <S>() => S;
 
   export function action<A extends Action>(action: A): ActionCmd<A>;
-  export function batch<A extends Action>(cmds: CmdType<A>[]): BatchCmd<A>;
-  export function sequence<A extends Action>(cmds: CmdType<A>[]): SequenceCmd<A>;
+  
+  export function list(
+    cmds: CmdType[],
+    options?: {
```

Comment:
same here with ways  to consolidate list options

Reply:
@nweber-gh i would maybe just say `type ListOptions = {...}` so  you can reuse it in both overloads